resources/views/event/display_includes/event_block.blade.php

```diff
@@ -24,32 +24,26 @@
             @endif
 
             {{-- Participating --}}
-            @if(Auth::check() && $event->activity?->isParticipating(Auth::user()))
-                @if($event->activity->isOnBackupList(Auth::user()))
+            @if(Auth::check() && $event->activity?->myParticipationCount > 0)
```

Comment:
Why were these booleans changed to integers exactly? I think the old syntax makes more sense and is more readable. How is someone supposed to have multiple participations and multiple backup list participations?

Reply:
The old syntax did extra database calls for every rendered activity. Now it is loaded upfront. I could move this check to the controller, but then this syntax would just be moved. We need to know per activity if you have any tickets or participation, and counting them in the same query is the most performant vs running 4 or 5 queries per activity.